content/docs/introduction/comparison.md

```diff
@@ -246,41 +246,36 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensuapp.org/) is broadly speaking a more modern Nagios.
+[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
 
 ### Scope
 
-The same general scope differences as in the case of
-[Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
+Sensu checks adhere to the [Nagios check specification](https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html). Sensu checks operate on two different models: publish and subscribe or standalone. It is possible to silence not just individual checks, but entire subscriptions. 
 
-The primary difference is that Sensu clients [register themselves](https://sensuapp.org/docs/0.27/reference/clients.html#what-is-a-sensu-client),
-and can determine the checks to run either from central or local configuration.
-Sensu does not have a limit on the amount of perfData.
+The primary difference is that Sensu clients [register themselves](https://docs.sensu.io/sensu-core/latest/reference/clients)
```

Comment:
You're not comparing it to anything to have a difference.

Reply:
So lines #253-#255 are from the original comparison doc. Nothing has changed in terms of the content. In fact, I reverted the change that I'd made. 

The original comparison was to Nagios. IMO it would seem that if the original doc wasn't providing any substantive comparison, then this bit should be completely removed. I'm happy to remove line 255 if you feel like that makes a bit more sense. If not, do you have a suggestion for something more appropriate to put here?

Reply:
I think I might have gotten mixed up between different version of the change, it seems okay now.